envbuilder.go

```diff
@@ -394,6 +396,14 @@ func Run(ctx context.Context, options Options) error {
 		})
 	}
 
+	// temp move of all ro mounts
+	tempRemountDest := filepath.Join("/", MagicDir)
+	ignorePrefixes := []string{tempRemountDest, "/proc", "/sys"}
```

Comment:
Should these be configurable?
Will we only ever support UNIX-based builds?

Reply:
Kaniko is currently Linux-only to the best of my knowledge.